_includes/v3.1/release-notes/v3.1.0-release-notes.md

```diff
@@ -0,0 +1,55 @@
+06 April 2018
+
+#### Support for OpenStack
+Calico v3.1 reintroduces support for OpenStack. Existing users can upgrade their Calico OpenStack clusters to
```

Comment:
It might be nice, now that we are in MD files, if we could use the variables {{site.prodname}} and {{site.nodecontainer}} THanks!